src/components/commons/Button.tsx

```diff
@@ -36,26 +39,36 @@ export default function Button({
   const handleButtonClick = () => {
     if (onClick) {
       onClick();
-      return;
     }
-    window.open(href, "_blank", "noreferrer");
   };
 
   const transitionStyle = "transition-all ease-in duration-300";
   return (
-    <button
-      data-testid={`${testId}-button`}
-      type="button"
-      className={`flex items-center justify-center rounded-[28px] group border border-white-50 brand-bg-gradient-1 active:brand-bg-gradient-2 hover:border-transparent
-                  ${transitionStyle} ${btnPadding} ${customStyle ?? ""}`}
-      disabled={disabled}
+    <div
```

Comment:
what's the diff of using `div` compared to `button`?

Reply:
was unable to preview the URL that users are going with `<button/>` hence decided to go with custom div to set the role as button

Reply:
it should work, can you try again? and `<a>` should fill the entire button else the link is not visible in between the `<a>` and `<button>` 
<img width="267" alt="image" src="https://user-images.githubusercontent.com/40191153/203894043-efdd93b0-8185-4ca2-811f-97941980d77c.png">